<?xml version="1.0" encoding="UTF-8"?>
<!-- ===============================================
  Copyright (C) 2014 ADVENTURE Project
  All Rights Reserved
================================================ -->
<ui version="4.0">
    <class>SetupDialog</class>
    <widget class="QDialog" name="SetupDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>459</width>
                <height>331</height>
            </rect>
        </property>
        <property name="minimumSize">
            <size>
                <width>459</width>
                <height>331</height>
            </size>
        </property>
        <property name="maximumSize">
            <size>
                <width>459</width>
                <height>331</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>Setting</string>
        </property>
        <widget class="QCheckBox" name="checkClearView">
            <property name="geometry">
                <rect>
                    <x>30</x>
                    <y>90</y>
                    <width>121</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="text">
                <string>Transparency</string>
            </property>
        </widget>
        <widget class="QCheckBox" name="checkBackImage">
            <property name="geometry">
                <rect>
                    <x>30</x>
                    <y>130</y>
                    <width>150</width>
                    <height>23</height>
                </rect>
            </property>
            <property name="text">
                <string>Background image</string>
            </property>
        </widget>
        <widget class="QLabel" name="label_2">
            <property name="geometry">
                <rect>
                    <x>50</x>
                    <y>210</y>
                    <width>171</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="text">
                <string>Lower left corner(m):</string>
            </property>
        </widget>
        <widget class="QLabel" name="label_4">
            <property name="geometry">
                <rect>
                    <x>50</x>
                    <y>240</y>
                    <width>251</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="text">
                <string>Scale(map/background, m/pixel): </string>
            </property>
        </widget>
        <widget class="QDoubleSpinBox" name="dbSpinBackMagni">
            <property name="geometry">
                <rect>
                    <x>350</x>
                    <y>240</y>
                    <width>81</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Sans Serif</family>
                    <pointsize>8</pointsize>
                </font>
            </property>
            <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="decimals">
                <number>2</number>
            </property>
            <property name="minimum">
                <double>0.010000000000000</double>
            </property>
            <property name="maximum">
                <double>100.000000000000000</double>
            </property>
            <property name="singleStep">
                <double>0.100000000000000</double>
            </property>
            <property name="value">
                <double>1.000000000000000</double>
            </property>
        </widget>
        <widget class="QDoubleSpinBox" name="dbSpinBackMapX">
            <property name="geometry">
                <rect>
                    <x>260</x>
                    <y>210</y>
                    <width>81</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Sans Serif</family>
                    <pointsize>8</pointsize>
                </font>
            </property>
            <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="decimals">
                <number>1</number>
            </property>
            <property name="minimum">
                <double>-1000000000.000000000000000</double>
            </property>
            <property name="maximum">
                <double>1000000000.000000000000000</double>
            </property>
            <property name="singleStep">
                <double>10.000000000000000</double>
            </property>
        </widget>
        <widget class="QDoubleSpinBox" name="dbSpinBackMapY">
            <property name="geometry">
                <rect>
                    <x>350</x>
                    <y>210</y>
                    <width>81</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Sans Serif</family>
                    <pointsize>8</pointsize>
                </font>
            </property>
            <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="decimals">
                <number>1</number>
            </property>
            <property name="minimum">
                <double>-1000000000.000000000000000</double>
            </property>
            <property name="maximum">
                <double>1000000000.000000000000000</double>
            </property>
            <property name="singleStep">
                <double>10.000000000000000</double>
            </property>
        </widget>
        <widget class="QLabel" name="label_5">
            <property name="geometry">
                <rect>
                    <x>50</x>
                    <y>160</y>
                    <width>91</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="text">
                <string>Image file:</string>
            </property>
        </widget>
        <widget class="QPushButton" name="buttonOk">
            <property name="geometry">
                <rect>
                    <x>330</x>
                    <y>280</y>
                    <width>101</width>
                    <height>31</height>
                </rect>
            </property>
            <property name="text">
                <string>OK</string>
            </property>
            <property name="default">
                <bool>true</bool>
            </property>
        </widget>
        <widget class="QPushButton" name="buttonCancel">
            <property name="geometry">
                <rect>
                    <x>210</x>
                    <y>280</y>
                    <width>101</width>
                    <height>31</height>
                </rect>
            </property>
            <property name="text">
                <string>Cancel</string>
            </property>
            <property name="default">
                <bool>false</bool>
            </property>
        </widget>
        <widget class="QPushButton" name="buttonBackFile">
            <property name="geometry">
                <rect>
                    <x>410</x>
                    <y>180</y>
                    <width>21</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="text">
                <string>...</string>
            </property>
        </widget>
        <widget class="QDoubleSpinBox" name="dbSpinViewMapY">
            <property name="geometry">
                <rect>
                    <x>350</x>
                    <y>20</y>
                    <width>81</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Sans Serif</family>
                    <pointsize>8</pointsize>
                </font>
            </property>
            <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="decimals">
                <number>1</number>
            </property>
            <property name="minimum">
                <double>-1000000000.000000000000000</double>
            </property>
            <property name="maximum">
                <double>1000000000.000000000000000</double>
            </property>
            <property name="singleStep">
                <double>10.000000000000000</double>
            </property>
        </widget>
        <widget class="QLabel" name="label_3">
            <property name="geometry">
                <rect>
                    <x>30</x>
                    <y>20</y>
                    <width>221</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="text">
                <string>Lower left corner(m): </string>
            </property>
        </widget>
        <widget class="QDoubleSpinBox" name="dbSpinViewMagni">
            <property name="geometry">
                <rect>
                    <x>350</x>
                    <y>50</y>
                    <width>81</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Sans Serif</family>
                    <pointsize>8</pointsize>
                </font>
            </property>
            <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="decimals">
                <number>2</number>
            </property>
            <property name="minimum">
                <double>0.010000000000000</double>
            </property>
            <property name="maximum">
                <double>100.000000000000000</double>
            </property>
            <property name="singleStep">
                <double>0.100000000000000</double>
            </property>
            <property name="value">
                <double>1.000000000000000</double>
            </property>
        </widget>
        <widget class="QLabel" name="label_6">
            <property name="geometry">
                <rect>
                    <x>30</x>
                    <y>50</y>
                    <width>261</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="text">
                <string>Scale(view/map, m/pixel):</string>
            </property>
        </widget>
        <widget class="QDoubleSpinBox" name="dbSpinViewMapX">
            <property name="geometry">
                <rect>
                    <x>260</x>
                    <y>20</y>
                    <width>81</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Sans Serif</family>
                    <pointsize>8</pointsize>
                </font>
            </property>
            <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="decimals">
                <number>1</number>
            </property>
            <property name="minimum">
                <double>-1000000000.000000000000000</double>
            </property>
            <property name="maximum">
                <double>1000000000.000000000000000</double>
            </property>
            <property name="singleStep">
                <double>10.000000000000000</double>
            </property>
        </widget>
        <widget class="QCheckBox" name="checkShowId">
            <property name="geometry">
                <rect>
                    <x>210</x>
                    <y>90</y>
                    <width>111</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="text">
                <string>Show ID</string>
            </property>
        </widget>
        <widget class="QLineEdit" name="lineEditBackFile">
            <property name="geometry">
                <rect>
                    <x>50</x>
                    <y>180</y>
                    <width>361</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Sans Serif</family>
                    <pointsize>8</pointsize>
                </font>
            </property>
        </widget>
    </widget>
    <tabstops>
        <tabstop>buttonCancel</tabstop>
        <tabstop>dbSpinViewMapX</tabstop>
        <tabstop>dbSpinViewMapY</tabstop>
        <tabstop>dbSpinViewMagni</tabstop>
        <tabstop>checkClearView</tabstop>
        <tabstop>checkShowId</tabstop>
        <tabstop>checkBackImage</tabstop>
        <tabstop>buttonBackFile</tabstop>
        <tabstop>dbSpinBackMapX</tabstop>
        <tabstop>dbSpinBackMapY</tabstop>
        <tabstop>dbSpinBackMagni</tabstop>
        <tabstop>buttonOk</tabstop>
    </tabstops>
    <resources/>
    <connections/>
</ui>
